plf-tomcat-resources/src/main/resources/conf/server.template.xml

```diff
@@ -105,10 +102,10 @@
 
     <!-- A "Connector" represents an endpoint by which requests are received
          and responses are returned. Documentation at :
-         Java HTTP Connector: /docs/config/http.html (blocking & non-blocking)
+         Java HTTP Connector: /docs/config/http.html
          Java AJP  Connector: /docs/config/ajp.html
          APR (HTTP/AJP) Connector: /docs/apr.html
-         Define a non-SSL HTTP/1.1 Connector on port 8080
+         Define a non-SSL/TLS HTTP/1.1 Connector on port 8080
     -->
     <Connector address="0.0.0.0" port="8080" protocol="org.apache.coyote.http11.Http11NioProtocol"
```

Comment:
> hmm, not sur I'm convinced. Not adding it by default, because in some architectures, it may not be interesting...
I'm just afraid that the developers will request tomcat in http2 (using only h2c probably) as the production environnements will continue to use the http/1.1 protocol. 

> We should upgrade to Servlet 4 for this and use HAProxy if needed (supports HTTP as reverse proxy).
Does servlet 4 will not be supported only starting from tomcat 9 ?
Haproxy support http/2 only doing load balancing on layer 4, not via the http reverse proxy.
In load balancing on layer 7, http2 is not supported by haproxy.

> So when the customers/community will request it, they will have it. This could be the very first step ;-)
Not activating this on tomcat doesn't mean the http/2 can't be used by the users. It can be used with nginx for example but in this mode `browser<---h2--->nginx/LB<---http--->tomcat`

Anyway, like for my other comment, we should validate the tomcat upgrade before changing the configuration.